src/node/desktop/src/main/utils.ts

```diff
@@ -600,3 +603,36 @@ export function getNumericEnvVar(envVarName: string): number | undefined {
   }
   return undefined;
 }
+
+const TRAILING_SLASH_REGEX = /[\\/]+$/;
+
+/**
+ * Removes trailing slashes from a string, optionally keeping the starting slash
+ * or double-slash for UNC paths.
+ * @param str The string to remove trailing slashes from
+ * @param keepStartSlash Whether to keep the starting slash if it exists
+ * @returns The string with trailing slashes removed
+ */
+export function removeTrailingSlashes(str: string, keepStartSlash = false): string {
```

Comment:
IMHO we can remove `keepStartSlash`; we should only remove trailing slashes from strings which have some non-slash components within. From what I can see in the usages in this PR, that parameter is always set to `true`, other than in tests?

Reply:
yeah in our existing usages, I've set the `keepStartSlash` to true to cover cases where `HOME=/` or similar. I wasn't sure if we'd more widely use `removeTrailingSlashes()` in the future, with strings that aren't paths? So I added `keepStartSlash` so it could be more general-purpose. But maybe unnecessary, and we can keep it simple/specific to paths (including simplifying the logic as you mentioned below)